docs/run.js

```diff
@@ -97,29 +97,61 @@ function drawTableAndSetUpHandlers(processedData, map, scoreAsset) {
  * @param {number|string} index checkbox index, basis for id
  * @param {String} displayName checkbox display name
  * @param {div} parentDiv div to attach checkbox to
+ * @param {Object} colorFunction color data from the layer
  * @return {HTMLInputElement} the checkbox
  */
 function createNewCheckbox(index, displayName, parentDiv) {
   const newRow = document.createElement('div');
   newRow.className = 'checkbox-row';
+
   const newBox = document.createElement('input');
```

Comment:
A nice feature (follow-up?) would be if you moused over the layer name, it gave you further information. For the discrete case, that would be really useful, since it could tell you what each color meant. For continuous it could give range. Could also say column that was being displayed.

Reply:
Add a TODO for this feature?